src/main/java/it/pagopa/afm/marketplacebe/service/BundleService.java

```diff
@@ -753,18 +753,19 @@ private boolean isPaymentAmountRangeValid(
 
     /**
      * Verify if transferCategoryList overlaps the target one,
-     * if either idChannel or idChannelTarget is of type ONUS relative to the other, the transferCategoryList check is bypassed.
+     * if either idChannel or idChannelTarget is of type ONUS relative to the other,
+     * or if the bundle type is PRIVATE, the transferCategoryList check is skipped.
      */
     private boolean isTransferCategoryListValid(
             List<String> transferCategoryList,
             List<String> transferCategoryListTarget,
             String idChannel,
-            String idChannelTarget
+            String idChannelTarget,
+            BundleType bundleType
     ) {
-        if(transferCategoryListTarget == null || (transferCategoryList != null && transferCategoryList.stream().noneMatch(transferCategoryListTarget::contains))) {
+        if (transferCategoryListTarget == null || bundleType == BundleType.PRIVATE || (transferCategoryList != null && transferCategoryList.stream().noneMatch(transferCategoryListTarget::contains))) {
```

Comment:
The check should be done taking into account both the target and the source bundle